source/packages/modules.md

```diff
@@ -167,14 +173,85 @@ You can also use traditional CommonJS syntax with CoffeeScript.
 
 ## Modular application structure
 
+Use in your application `package.json` file the section `meteor`.
+
+> This is available since Meteor 1.7
+
+```json
+{
+  "meteor": {
+    "mainModule": {
+      "client": "client/main.js",
+      "server": "server/main.js"
+    }
+  }
+}
+```
+
+When specified, these entry points will define in which files Meteor is going to start the evaluation process for each architecture (client and server).
+
+This way Meteor is not going to eager load any other files.
+
+In addition to `meteor.mainModule`, the `meteor` section of `package.json` may also specify `meteor.testModule` to control which test modules are loaded by `meteor test` or `meteor test --full-app`:
+
+```json
+{
+  "meteor": {
+    "mainModule": {
+      "client": "client/main.js",
+      "server": "server/main.js"
+    },
+    "testModule": "tests.js"
+  }
+}
+```
+
+If your client and server test files are different, you can expand the testModule configuration using the same syntax as mainModule:
+
+```json
+{
+  "meteor": {
+    "mainModule": {
+      "client": "client/main.js",
+      "server": "server/main.js"
+    },
+    "testModule": {
+      "client": "client/tests.js",
+      "server": "server/tests.js"
+    }
+  }
+}
+```
+
+The same test module will be loaded whether or not you use the `--full-app` option. 
+
+Any tests that need to detect `--full-app` should check `Meteor.isAppTest`. 
```

Comment:
I think there should be a link here to the testing section that explains this in a bit more details.